common/Perf/Azure.Test.Perf/NumberFormatter.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Test.Perf
```

Comment:
Definitely shouldn't be picking these up in the PR.   You'll most likely need to rebase to `main` in order to ensure that the only files here are those you're intending to change.